src/config/firefox-desktop.js

```diff
@@ -12,47 +12,67 @@ import { validate, noResponse } from '../utils/data-validation';
 export default {
   sampleRate: 0.1,
   dimensions: {
-    os: {
-      title: 'OS',
-      key: 'os',
-      values: [
-        { key: '*', label: 'All OSes' },
-        { key: 'Windows', label: 'Windows' },
-        { key: 'Mac', label: 'Mac' },
-        { key: 'Linux', label: 'Linux' },
-      ],
-    },
-    channel: {
-      title: 'Channel',
-      key: 'channel',
-      values: [
-        { key: 'nightly', label: 'Nightly' },
-        { key: 'beta', label: 'Beta' },
-        { key: 'release', label: 'Release' },
-      ],
-      isValidKey(key, probe) {
-        return (probe.prerelease && key !== 'release') || !probe.prerelease;
+    productDimensions: {
```

Comment:
Can you give more details why we have `dimensions.productDimensions` and not just `dimensions` here? This seems like an unnecessary nesting

Reply:
We hashed this out on Slack